usr/share/rear/conf/default.conf

```diff
@@ -1416,14 +1445,7 @@ BACKUP_PROG_CRYPT_ENABLED="false"
 # cf. the reasoning about SSH_UNPROTECTED_PRIVATE_KEYS below
 # and see https://github.com/rear/rear/issues/2155
 # Therefore BACKUP_PROG_CRYPT_KEY must be manually set before running "rear recover".
-# Avoid that the BACKUP_PROG_CRYPT_KEY value is shown when usr/sbin/rear was called with 'set -x'
-# for debugging usr/sbin/rear cf. https://github.com/rear/rear/issues/2144#issuecomment-493908133
-# and see the comment of the UserInput function in lib/_input-output-functions.sh
-# how to keep things confidential when usr/sbin/rear is run in debugscript mode.
-# In debugscript mode only scripts sourced by the Source function in lib/framework-functions.sh
-# are run with 'set -x' but default.conf is sourced by usr/sbin/rear directly.
-# Nevertheless the confidential way via { confidential_command ; } 2>/dev/null
-# is also used here to provide a template how to do it properly in etc/rear/local.conf:
+# In local.conf set it confidentially via { BACKUP_PROG_CRYPT_KEY='secret_key' ; } 2>/dev/null
```

Comment:
```suggestion
```

Reply:
Since my https://github.com/rear/rear/pull/2156
I consider `BACKUP_PROG_CRYPT_KEY` to some extent as
a config variable where I became especially responsible for
so I left this special additional explanation for
this variable according to my personal reasoning in
https://github.com/rear/rear/pull/2982#discussion_r1189910601
while in contrast I consider `GALAXY11_PASSWORD` more
as "your variable" because you made the GALAXY11 code
which is why I removed this special additional explanation
for "your variable" according to your request in
https://github.com/rear/rear/pull/2982#discussion_r1189888998